src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public class ColorSupplier {
+    public static String getRandomColor() {
+        String[] colors = {"red", "blue", "white", "black", "pink", "green"};
```

Comment:
```suggestion
public class ColorSupplier {
	private String[] colors = {"red", "blue", "white", "black", "pink", "green"};

    public String getRandomColor() {
```
it's better to make this field a class variable. This way we won't create new object of color per each method call